docs/fast_data/runtime_management/secure_access.md

```diff
@@ -0,0 +1,2934 @@
+        "value": "noopen"
+      },
+      "append": false
+    },
+    {
+      "header": {
+        "key": "Referrer-Policy",
+        "value": "strict-origin-when-cross-origin"
+      },
+      "append": false
+    }
+  ]
+```
+
+---
+
+:::note
+In case your project does not employ Envoy as API Gateway, please remember to adapt your API Gateway configuration
+accordingly to support the runtime management system features, such as [_Websocket connection upgrade_](https://developer.mozilla.org/en-US/docs/Web/HTTP/Protocol_upgrade_mechanism#upgrading_to_a_websocket_connection).
+:::
+
+## Authorization
+
+In this section are described which resources should be configured in order to properly grant the access
+to the Fast Data Runtime Management solution only to authorized users.
+
+### Services
+
+#### Authorization Service
+
+This service is introduced with the application creation and it is already preconfigured with all the necessary configs.
+In case the service already existed in your project, please ensure the following environment variable on the service contains
+the following values
+
+| Environment Variable | Value                                                                                                                                          |
+|----------------------|------------------------------------------------------------------------------------------------------------------------------------------------|
+| HEADERS_TO_PROXY     | connection,upgrade,sec-websocket-protocol,sec-websocket-version,x-request-id,request-id,cookie,authorization,client-type,host,x-forwarded-host |
+| USERINFO_URL         | http://authentication-service/userinfo                                                                                                         |
+
+In order to support real-time updates in the Fast Data Runtime Management system, it is important to allow proxying the following
+headers:
+- [`connection`](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Connection)
+- [`upgrade`](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Upgrade)
+- [`sec-websocket-protocol`](https://developer.mozilla.org/en-US/docs/Web/HTTP/Protocol_upgrade_mechanism#websocket-specific_headers)
+- [`sec-websocket-version`](https://developer.mozilla.org/en-US/docs/Web/HTTP/Protocol_upgrade_mechanism#websocket-specific_headers)
+
+#### Rönd
+
+Once all the services are ready, it is necessary to protect them. To achieve so, in this guide we are going to rely on [Rönd](https://rond-authz.io/),
+which allows to define for each service's endpoints the proper access policies. Hence, let's enable Rönd sidecar for the following
+two services:
+
+- `control-plane` → to regulate who can retrieve Fast Data runtime configuration and change its runtime status
+- `crud-service` → to regulate who can access roles and bindings (discussed later)
+
+An example of Rönd enabled for these services can be observed in the image:
```

Comment:
Here we can mention that enabling Rond is possible through the Authorization section in Console Design Area (eventually with also a link to redirect to that docs page)